pytests/func/test_output.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root
+# for license information.
+
+from __future__ import print_function, with_statement, absolute_import
+from pytests.helpers.session import DebugSession
+
+
+def test_with_no_output(pyfile, run_as, start_method):
+    @pyfile
+    def code_to_debug():
+        from dbgimporter import import_and_enable_debugger
+        import_and_enable_debugger()
+        # Do nothing, and check if there is any output
+
+    with DebugSession() as session:
+        session.initialize(target=(run_as, code_to_debug), start_method=start_method)
+        session.start_debugging()
+        session.wait_for_exit()
+        assert b'' == session.get_stdout_as_string()
+        assert b'' == session.get_stderr_as_string()
+
+
+def test_with_tab_in_output(pyfile, run_as, start_method):
+    @pyfile
+    def code_to_debug():
+        from dbgimporter import import_and_enable_debugger
+        import_and_enable_debugger()
+        print('Hello\tWorld')
+
+    with DebugSession() as session:
+        session.initialize(target=(run_as, code_to_debug), start_method=start_method)
+        session.start_debugging()
+        session.wait_for_exit()
+        data = session.get_stdout_as_string() + session.get_stderr_as_string()
```

Comment:
We probably want to check either channel separately. You can do `print(file=sys.stderr)` in the test to exercise stderr.

Or is there some outstanding issue where they get mixed up?

Reply:
There is no mixup. Just checking stdout should be enough.

Reply:
Shouldn't we check stderr too tho, just to make sure it gets captured as well?

Reply:
Actually this is wrong. This should check the output event.

Reply:
This test is specifically for tab character. There was an odd behavior with that one. I will add more output tests and cover stderr output there